koku/api/metrics/views.py

```diff
@@ -15,19 +15,27 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 """Views for CostModelMetricsMap."""
+import copy
 import logging
+import os
 
+from django.utils.encoding import force_text
 from django.views.decorators.vary import vary_on_headers
 from rest_framework import mixins
+from rest_framework import status
 from rest_framework import viewsets
+from rest_framework.exceptions import APIException
 from rest_framework.permissions import AllowAny
 
 from api.common import RH_IDENTITY_HEADER
-from api.metrics.models import CostModelMetricsMap
+from api.metrics import constants as metric_constants
 from api.metrics.serializers import CostModelMetricMapSerializer
+from koku.settings import BASE_DIR
 
 LOG = logging.getLogger(__name__)
 
+COST_MODEL_METRICS_FILE_NAME = os.path.join(BASE_DIR, "api/metrics/data/cost_models_metric_map.json")
```

Comment:
As far as I can tell I don't think we need this any more and can also remove the `/metrics/data/cost_models_metric_map.json` file

Reply:
removed